### ISSUE TITLE
Differenziertere Benutzerrechte für Themes

### ISSUE THREAD

**Issue by Mynyx** [contao/contao#871]
Analog dazu wie es bei anderen Backend-Modulen bereits möglich ist, wäre es auch bei Themes sinnvoll, wenn für Benutzer bzw. Benutzergruppen "Erlaubte Themes" sowie "Theme-Rechte" für das Anlegen und Löschen konfiguriert werden könnten.

Derzeit könnte ein Benutzer, der beispielsweise lediglich das Recht für das Backend-Modul Themes und ein einzelnes Theme-Modul hat, auch alle Themes löschen.

**Comment by aschempp** [comment]
wurden in 4.9 nicht die Theme Import- und Export-Optionen angepasst?

**Comment by leofeyer** [comment]
Nein, das haben wir zwar in #853 diskutiert, aber letztendlich haben wir dann #1196 implementiert.

**Comment by Grasuzu** [comment]
Aber wenn ich mich nicht irre, löst das das Problem feinerer Benutzerrechte nicht, oder? 
Eine Art Pagemount für "Erlaubte Themes" fehlt mir gerade.
